src/routes/email/email.controller.delete-unverified-emails.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { AppModule } from '@/app.module';
+import { CacheModule } from '@/datasources/cache/cache.module';
+import { TestCacheModule } from '@/datasources/cache/__tests__/test.cache.module';
+import configuration from '@/config/entities/__tests__/configuration';
+import { RequestScopedLoggingModule } from '@/logging/logging.module';
+import { TestLoggingModule } from '@/logging/__tests__/test.logging.module';
+import { NetworkModule } from '@/datasources/network/network.module';
+import { TestNetworkModule } from '@/datasources/network/__tests__/test.network.module';
+import { TestAppProvider } from '@/__tests__/test-app.provider';
+import { EmailDataSourceModule } from '@/datasources/email/email.datasource.module';
+import { TestEmailDatasourceModule } from '@/datasources/email/__tests__/test.email.datasource.module';
+import { faker } from '@faker-js/faker';
+import { IEmailDataSource } from '@/domain/interfaces/email.datasource.interface';
+import { chainBuilder } from '@/domain/chains/entities/__tests__/chain.builder';
+import { EmailControllerModule } from '@/routes/email/email.controller.module';
+import { safeBuilder } from '@/domain/safe/entities/__tests__/safe.builder';
+import { Email, EmailAddress } from '@/domain/email/entities/email.entity';
+
+const oneWeekInMs = 7 * 24 * 60 * 60 * 1_000;
+
+describe('Email controller delete unverified emails tests', () => {
+  let app;
+  let emailDatasource;
+
+  beforeEach(async () => {
+    jest.clearAllMocks();
+    jest.useFakeTimers();
+
+    // Temporarily allow us to alter timezone calculation for testing
+    // TODO: Remove after setting Jest timezone to UTC
+    Date.prototype.getTimezoneOffset = jest.fn(() => 0);
```

Comment:
We should move this to a global setup for jest. Sadly it placing it in our `e2e-setup.ts` doesn't work. I think we should approach this in a separate PR, however.